src/AutoRest.CSharp.V3/Output/Models/Requests/Method.cs

```diff
@@ -22,7 +22,7 @@ public Method(string name, string? description, Request request, Parameter[] par
         public string? Description { get; }
         public Request Request { get; }
         public Parameter[] Parameters { get; }
-        public Response Response { get; }
+        public Response Response { get; set; }
```

Comment:
Why is this mutable?

Reply:
Required to mutate to modify the original method's response when determining LRO operations. Original methods only return Response (not Response<T>). You suggested I use this implementation as to simplify how we handle LRO when looping over the operations.